### PR TITLE
Run libreoffice

### PR DESCRIPTION
**Pull request by Floppy**
Getting libreoffice running...

### CODE REVIEW

**Comment by Floppy** [comment]
On AWS, I had to do 3 things:

* Changed handler to `ds-caselaw-pdf-conversion.lambda_function.handler`
* Changed RAM to 512MB
* Changed timeout to 1 minute

**Comment by Floppy** [comment]
Currently running, but terminating with `com::sun::star::container::NoSuchElementException`

**Comment by Floppy** [comment]
This is now working using a gzipped rebuild of the brotli layer. I've uploaded it at https://eu-west-2.console.aws.amazon.com/lambda/home?region=eu-west-2#/layers/libreoffice-without-brotli/versions/1?tab=versions. We'll need to copy it over to production at some point

**Comment by Floppy** [comment]
https://github.com/shelfio/libreoffice-lambda-layer/issues/24 is the underlying issue with the gzip layers, which seem to be broken.

File: scripts/setup-localstack.sh [hunk 1]
```diff
@@ -11,6 +11,9 @@ awslocal lambda create-function \
   --runtime python3.9 \
   --environment "Variables={PUBLIC_ASSET_BUCKET=$PUBLIC_ASSET_BUCKET,AWS_SECRET_KEY=$AWS_SECRET_KEY,AWS_ACCESS_KEY_ID=$AWS_ACCESS_KEY_ID,AWS_ENDPOINT_URL=$AWS_ENDPOINT_URL}" \
   --role arn:aws:iam::000000000000:role/lambda-role \
+  --layers arn:aws:lambda:eu-west-2:764866452798:layer:libreoffice-gzip:1 \
+  --memory-size 512 \
+  --timeout 600
```
Comment (dragon-dxw) [inline scripts/setup-localstack.sh:16]: This timeout may be excessive (it's 10 minutes, we set it high for diagnostic reasons)

File: Makefile [hunk 2]
```diff
@@ -14,5 +14,10 @@ update:
 	make build
 	@sh scripts/update-lambda.sh
 
+lambda:
```
Comment (dragon-dxw) [inline Makefile:17]: We should probably document the build/upload process better.

**Review by dragon-dxw — COMMENTED** [review]
We'll need to squash all the commits, but so happy this is working.

**Pull request closed without merging**